### ISSUE TITLE
Test modules uninstallation independently of WordPoints uninstallation

### ISSUE THREAD

**Issue by JDGrimes** [WordPoints/module-uninstall-tester#10]
After #9 we now test module uninstallation by uninstalling WordPoints as a whole. This has the benefit that it allows us to check that the module includes any dependencies that it needs beyond what WordPoints includes during uninstall. This is important, because otherwise a module might assume that all of WordPoints will always be loaded when it is uninstalled, but that is only true when the module is being uninstalled independently of WordPoints.

However, it is also important to test uninstalling the module independently as well, because otherwise we don't know whether it properly cleans up after itself, or whether it is just that WordPoints is cleaning up after it.

So really what we need here is to run the test both ways: once uninstalling WordPoints as a whole, and once just uninstalling the module.

**Comment by JDGrimes** [comment]
Closing in favor of https://github.com/WordPoints/dev-lib/issues/192.
